### PR TITLE
Automate configuration generation

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

File: Scripts/altapay_config.php [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            $this->resourceConfig->saveConfig(
+                'payment/terminal' . $i . '/active',
+                1,
+                'default',
+                0
+            );
+
+            $this->resourceConfig->saveConfig(
+                'payment/terminal' . $i . '/title',
+                $terminals[$i],
```
Comment (emicha) [inline Scripts/altapay_config.php:130]: This does not seem to be generic. What happens if the terminal list is more or less than 5 terminals?

Comment (BushraAsif) [reply Scripts/altapay_config.php:130]: In Magento 2 we can only set 5 terminals from the configuration that's why we put this loop in here.

Comment (emicha) [reply Scripts/altapay_config.php:170]: @BushraAsif what if the number of terminals available is only 4? This will fail

Comment (BushraAsif) [reply Scripts/altapay_config.php]: @emicha, This has been updated.

**Review by emicha — APPROVED** [review]

**Pull request merged**